.gitignore

```diff
@@ -18,6 +18,7 @@ idea
 *.sage.py
 params
 _build
+build
```

Comment:
my IDE automatically ran cmake (I think?) and added this folder, so let's gitignore it